biomodels/src/main/java/org/reactome/release/BioModelsUtilities.java

```diff
@@ -0,0 +1,105 @@
+        GKInstance biomodelsReferenceDatabase = retrieveBioModelsDatabaseInstance(dba);
+
+        if (biomodelsReferenceDatabase == null) {
+            logger.info("Creating BioModels reference database - no existing one was found");
+            biomodelsReferenceDatabase = createBioModelsDatabaseInstance(dba, instanceEdit);
+        }
+
+        return biomodelsReferenceDatabase;
+    }
+
+    /**
+     * Attempts to find the biomodels database instance.
+     * @param dba
+     * @return -- (GKInstance) The biomodels database instance, if it was found. Returns null if not.
+     */
+    public static GKInstance retrieveBioModelsDatabaseInstance(MySQLAdaptor dba) {
+
+        GKInstance biomodelsReferenceDatabase = null;
+        try {
+            @SuppressWarnings("unchecked")
+            Collection<GKInstance> biomodelsReferenceDatabaseCollection = dba.fetchInstanceByAttribute(
+                    ReactomeJavaConstants.ReferenceDatabase,
+                    ReactomeJavaConstants.name,
+                    "=",
+                    "BioModels");
+            // TODO: Once merged into develop, this collection null check can be handled by 'safeList' in CollectionUtils
+            if (biomodelsReferenceDatabaseCollection != null && biomodelsReferenceDatabaseCollection.size()  != 0) {
+                biomodelsReferenceDatabase = biomodelsReferenceDatabaseCollection.iterator().next();
+            }
+        } catch (Exception e) {
+            logAndThrow("Unable to retrieve BioModels reference database", e);
+        }
+        return biomodelsReferenceDatabase;
+    }
+
+    /**
+     * Creates biomodels database instance.
+     * @param dba
+     * @param instanceEdit
+     * @return -- (GKInstance) The biomodels database instance
+     */
+    private static GKInstance createBioModelsDatabaseInstance(MySQLAdaptor dba, GKInstance instanceEdit) {
+
+        GKInstance biomodelsReferenceDatabase;
+        biomodelsReferenceDatabase = new GKInstance();
+        biomodelsReferenceDatabase.setDbAdaptor(dba);
+        SchemaClass referenceDatabase = dba.getSchema().getClassByName(ReactomeJavaConstants.ReferenceDatabase);
+        biomodelsReferenceDatabase.setSchemaClass(referenceDatabase);
+
+        try {
+            biomodelsReferenceDatabase.addAttributeValue(ReactomeJavaConstants.created, instanceEdit);
+            String bioModelsDatabaseName = "BioModels Database";
+            biomodelsReferenceDatabase.addAttributeValue(
+                    ReactomeJavaConstants.name, Arrays.asList(bioModelsDatabaseName, "BioModels")
+            );
+            biomodelsReferenceDatabase.setDisplayName(bioModelsDatabaseName);
+
+            //TODO Check if both accessUrl and Url should be using https
+            biomodelsReferenceDatabase.addAttributeValue(
+                    ReactomeJavaConstants.accessUrl, "http://www.ebi.ac.uk/biomodels-main/publ-model.do?mid=###ID###"
```

Comment:
I just got an e-mail from Peter and Lisa and they found the BioModels access URL is now outdated.  It looks like this is the new one: https://www.ebi.ac.uk/biomodels/###ID### (e.g. https://www.ebi.ac.uk/biomodels/BIOMD0000000007)